spec/rubocop/cop/layout/empty_lines_around_attribute_accessor_spec.rb

```diff
@@ -42,6 +42,15 @@ def do_something
     RUBY
   end
 
+  it 'accepts code that separates a attribute accessor from the code and comment line with a newline' do
+    expect_no_offenses(<<~RUBY)
+      attr_accessor :foo
+      # comment
+      def do_something
+      end
+    RUBY
```

Comment:
#10693 reports a false positive in the following cases:

```ruby
    # Disable cop to allow comments on each accessor in a separate line
    # rubocop:disable Style/AccessorGrouping
    attr_accessor :var      # name of global variable to assign it to if any
    attr_accessor :points   # array of [lat, long]
    attr_accessor :color    # css-format color string
    attr_accessor :weight   # width of line in pixels
    attr_accessor :opacity  # from 0.0 to 1.0
    # rubocop:enable Style/AccessorGrouping

    more code
```

This added test case looks like it should be `expect_offense` instead of `expect_no_offenses `.

Reply:
Indeed is it, I fixed it.